### PR TITLE
Remove TopKnormalizer from MO IR Reader transformation_list

### PR DESCRIPTION
**Pull request by achetver**
Code:

[x] Comments

[x] Code style (PEP8)

[x] Transformation generates reshape-able IR

[x] Transformation preserves node names

Validation:

[x] Unit tests: N/A 

[x] Framework layer tests: N/A - no new layers were enabled in the MO

[x] Transformation tests: N/A - no new transformations implemented

 [x] MO IR Reader check: Manual check done

### CODE REVIEW

**Comment by jane-intel** [comment]
Please rebase. Stress test fixes were merged recently.

**Comment by lazarevevgeny** [comment]
> Please rebase. Stress test fixes were merged recently.

The rebase will not help. According to the tests results the transformation is needed. Let's understand the root cause of the issue why this transformation was needed for a failed model.

**Comment by achetver** [comment]
This transformation was needed to fix missed output result nodes for TopK node. topk_add_output_result function added to graph restoring process to fix this issue.

File: model-optimizer/mo/utils/ir_reader/layer_to_class.py [hunk 1]
```diff
@@ -233,6 +233,21 @@ def assign_add_output_result(op: Node):
     op.out_port(0).connect(tmp_result.in_port(0))
 
 
+def topk_add_output_result(op: Node):
```
Comment (lazarevevgeny) [inline model-optimizer/mo/utils/ir_reader/layer_to_class.py:236]: The code of this function is identical to the code in the TopKNormalizer. Let's update the TopKNormalizer transformation and this function to it. Also update the IR Reader to use the function from new location.

File: model-optimizer/mo/utils/ir_reader/layer_to_class.py [hunk 2]
```diff
@@ -233,6 +233,21 @@ def assign_add_output_result(op: Node):
...
+    """
+    Function adds necessary output result node for TopK node
```
Comment (lazarevevgeny) [inline model-optimizer/mo/utils/ir_reader/layer_to_class.py:238]: Let's make the comment more detailed and describe all input/output values.

File: model-optimizer/mo/utils/ir_reader/layer_to_class.py [hunk 3]
```diff
@@ -233,6 +233,21 @@ def assign_add_output_result(op: Node):
...
+    :param op:
+    :return:
+    """
+    if op.out_port(0).disconnected():
```
Comment (lazarevevgeny) [inline model-optimizer/mo/utils/ir_reader/layer_to_class.py:242]: nit: may be it makes sense to make the function more generic. If not then please, add an assert in the beginning that the node.soft_get('op') == 'TopK'

File: model-optimizer/mo/utils/ir_reader/layer_to_class.py (hunk 1 above)
Comment (lazarevevgeny) [inline model-optimizer/mo/utils/ir_reader/layer_to_class.py:236]: Please, change the name of the variable from `op` to `node`

Comment (achetver) [reply model-optimizer/mo/utils/ir_reader/layer_to_class.py:237]: Normalizer and reader function updated.

File: model-optimizer/mo/utils/ir_reader/layer_to_class.py (hunk 3 above)
Comment (achetver) [reply model-optimizer/mo/utils/ir_reader/layer_to_class.py]: Assert added.

File: model-optimizer/mo/utils/ir_reader/layer_to_class.py (hunk 1 above)
Comment (achetver) [reply model-optimizer/mo/utils/ir_reader/layer_to_class.py:237]: Do we really need this change? We use `op` name inside all functions in this file.

File: model-optimizer/mo/utils/ir_reader/layer_to_class.py (hunk 2 above)
Comment (achetver) [reply model-optimizer/mo/utils/ir_reader/layer_to_class.py]: Comments updeted.

File: model-optimizer/mo/utils/ir_reader/layer_to_class.py (hunk 1 above)
Comment (lazarevevgeny) [reply model-optimizer/mo/utils/ir_reader/layer_to_class.py:237]: Remove this function completely. Just use it `TopKNormalizer.normalize_outputs` in line 275.

**Comment by lazarevevgeny** [comment]
@achetver , the pre-commit failed. Please, restart.

File: model-optimizer/extensions/back/TopKNormalizer.py [hunk 4]
```diff
@@ -58,6 +58,16 @@ def replace_pattern(graph: Graph, match: dict):
                                                    {'override_output_shape': True})
         node.in_port(1).get_connection().insert_node(reshape)
 
+        TopKNormalizer.normalize_outputs(node, graph)
+
+    @staticmethod
+    def normalize_outputs(node: Node, graph: Graph = None):
```
Comment (lazarevevgeny) [inline model-optimizer/extensions/back/TopKNormalizer.py:64]: Remove the `graph` parameter. Get the `graph` from the `node`.

Comment (achetver) [reply model-optimizer/extensions/back/TopKNormalizer.py]: Done.

**Pull request merged**